reana_cluster/backends/kubernetes/templates/clusterrole/reana-system.yaml

```diff
@@ -3,7 +3,7 @@ kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   namespace: default
-  name: service-reader
+  name: batch-interactive-manager
```

Comment:
"batch" and "interactive" are all deployments we do... So perhaps shorten to "deployment-manager" and "manage-deployments" later?